lib/versioning/node/schedule.ts

```diff
@@ -14,3 +15,26 @@ export const nodeSchedule: NodeJsData = JSON.parse(
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   dataFiles.get('data/node-js-schedule.json')!
 );
+
+export type NodeJsScheduleWithVersion = { version: string } & NodeJsSchedule;
+
+export function findScheduleForCodename(
+  codename: string
+): NodeJsScheduleWithVersion | null {
+  for (const version of Object.keys(nodeSchedule)) {
```

Comment:
I think we should create a static `Map<string,NodeJsScheduleWithVersion>` once and use that as lookup. That will speedup lookup as we can fully ignore non lts versions.

Reply:
Makes sense! Done with https://github.com/renovatebot/renovate/pull/13466/commits/a2f0211cdc4ffc7f39323398d77d46cc9e638533.